src/java.logging/share/conf/logging.properties

```diff
@@ -43,7 +43,7 @@ java.util.logging.FileHandler.count = 1
 java.util.logging.FileHandler.maxLocks = 100
 java.util.logging.FileHandler.formatter = java.util.logging.XMLFormatter
 
-# Limit the message that are printed on the console to INFO and above.
+# Limit the messages that are printed on the console to INFO and above.
```

Comment:
"Limit the messages ..." could be more direct. Consider "Only print messages to the console if they have level INFO or above."

Reply:
Understood and thanks. That said, I will not use what you suggested for the sake of consistency: the "limit the messages" phrasing is also used in the API:

```
    /**
     * Set the log level specifying which message levels will be
     * logged by this {@code Handler}.  Message levels lower than this
     * value will be discarded.
     * <p>
     * The intention is to allow developers to turn on voluminous
     * logging, but to limit the messages that are sent to certain
     * {@code Handlers}.
     *
     * @param newLevel   the new value for the log level
     * @throws  SecurityException  if a security manager exists and if
     *             the caller does not have {@code LoggingPermission("control")}.
     */
    public synchronized void setLevel(Level newLevel) throws SecurityException {
```